python/lsst/ts/ATPneumaticsSimulator/__init__.py

```diff
@@ -20,4 +20,12 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 from .pneumatics_csc import *
-from .version import *  # Generated by sconsUtils
+
+try:
+    from .version import *  # Generated by sconsUtils
+except ModuleNotFoundError:
+    __version__ = "unknown"
+    __repo_version__ = "unknown"
+    __fingerprint__ = "unknown"
+    __dependency_versions__ = {
```

Comment:
Do you really need all these extra fields? Why? We have a lot of packages to clean up if we need all this stuff.